corehq/apps/events/tasks.py

```diff
@@ -71,7 +71,7 @@ def get_user_attendee_cases_on_domain(domain):
 
 
 def get_case_block_for_user(user, owner_id, attendee_case_type):
-    case_name = ' '.join((user.first_name, user.last_name))
+    case_name = user.username
```

Comment:
There's a function to remove the "@root.commcarehq.org" part (similar to how its shown in the mobile workers list). I'll quickly try and find it.

Reply:
I was thinking of [this](https://github.com/dimagi/commcare-hq/blob/0f618f785a5feb6932e42ee6a5f111b5162caf62/corehq/apps/userreports/transforms/custom/users.py#L12) function, but it takes a `user_id`, so maybe write another one.

Reply:
actually scratch that, the MW page shows only the username. Ima update